test/test_bounds.c

```diff
@@ -31,6 +31,7 @@
 START_TEST(test_bounds)
 {
    opt_set_int(OPT_MISSING_BODY, 0);
+   opt_set_int(OPT_MISSING_WAIT, 0);
```

Comment:
As there's so many of these it would be better to do it once in `main()` in `test/unit_test.c` and then turn it on for the one test that needs it.

Reply:
That's a nicer way to do it for sure. 😅